dali/pipeline/data/tensor_list.h

```diff
@@ -63,11 +63,17 @@ class DLL_PUBLIC TensorList : private Buffer<Backend> {
 
   DLL_PUBLIC ~TensorList() = default;
 
+  /**
+   * @brief Number of elements in Tensor List.
+   */
+  int64_t tl_elements() const {
```

Comment:
This name looks too much like number of tensor _list elements_ - i.e. num_samples. Suggested alternative:
```suggestion
  int64_t num_elements() const {
```
which is what we have in `TensorListView`.

Reply:
I will be changing that in the next PRs, so I want to have some unique name for it as mentioned in the PR description. Otherwise it's rather cumbersome to find all the occurrences.